templates/workflows-kms-key.yaml

```diff
@@ -21,8 +29,12 @@ Resources:
           - - !Sub 'arn:aws:iam::${AWS::AccountId}:root'
             - !ImportValue
               'Fn::Sub': '${AWS::Region}-workflows-nextflow-ci-service-account-ServiceUserArn'
```

Comment:
service user should probably only be allowed to `use` the key not admin it?

Reply:
@zaro0508 Doesn't the server user have to be able to admin it in order to create and delete through CI?

Reply:
ahh, yes you are correct. Service user needs access to update it once it creates it.